### ISSUE TITLE
New Quest: Can you pay using Swish here (Sweden only)

### ISSUE THREAD

**Issue by squeak9** [streetcomplete/StreetComplete#5043]
Swish is an app based payment method that is widely used by over 80% of the population of Sweden, it lets you send money to other users using only their phone number but you can also send it to the number of a store to use as payment, scan a QR code at the register or use NFC or Bluetooth on your phone to initiate a payment. It has become a replacement for cash in many ways.

More info: https://en.wikipedia.org/wiki/Swish_(payment)

### General
Affected tag(s) to be modified/added: [payment:swish](https://wiki.openstreetmap.org/wiki/Key:payment:swish)
Question asked: **Can you pay using Swish here?**

### Checklist
Checklist for quest suggestions (see [guidelines](https://github.com/streetcomplete/StreetComplete/blob/master/QUEST_GUIDELINES.md)):
- [x] 🚧 To be added tag is established and has a useful purpose
The tag isn't super common right now sadly but it is established and useful. Once more data starts flowing in then apps might pick it up. I can speak from experience that if you lose your wallet but not your phone then this data is invaluable..
- [x] 🤔 Any answer the user can give must have an equivalent tagging (Quest should not reappear to other users when solved by one)
It's just `yes` or `no` or possibly `only`.
- [x] 🐿️ Easily answerable by any pedestrian from the outside but a survey is necessary
Some stores might have [signs outside](https://www.swish.nu/marketing-toolbox?lang=en) that say that indicate that they accept Swish but if they don't you can often see a sign by the register or ask if you're a customer.
It's as easy as the other payment method quests at least.
- [x] 💤 Not an overwhelming percentage of quests have the same answer (No spam)
- [x] 🕓 Applies to a reasonable number of map data (Worth the effort)
The tag is potentially very useful because there are over 8 million Swish users in Sweden (population of 10 million) and it's getting very common that stores and restaurants accept it as payment. Over [350 000 companies](https://www.swish.nu/about-swish) let you pay using Swish, a fair number of those might be online stores but enough should be physical. For example Coop which is one of the biggest grocery store chains is [starting to roll out](https://kundservice.coop.se/hc/sv/articles/5326578021138-Swish-i-butik-) Swish payment in some of their stores. You also see it as payment for vending machines or public bathrooms.

### Ideas for implementation

**Element selection:** 
Shops, restaurants and other places where you might pay money but only in Sweden.
Maybe it would be nice if it could ask again later since the accepted payment methods might change but that would also be true with the cash-quest.. might be a separate issue?

**Metadata needed:** 

**Proposed UI:**
Basically the same as the cash question, just a `yes` or `no` button.

**Comment by mnalis** [comment]
Related (choosing different payment options) especially this comment with pictures (but also the rest of discussion) https://github.com/streetcomplete/StreetComplete/issues/3219#issuecomment-922095785 .

**Comment by westnordost** [comment]
I am sorry, but I've now decided that any tagging of `payment:*` where * is a specific payment service, from bitcoin, to paypal, to google pay, to mastercard, to debit card, shall not be done with this app. Generally.

The reasons are:

- **💵 Valuable surveyors:** Whether or not a shop accepts certain payment methods is a very volatile information that may change frequently or even without the shop's knowing. Volatile data necessitates more frequent re-checking which further draws on the limited resource of OpenStreetMap surveyors time.

- **🕵️ Publicly accessible on foot:** Especially non-hype currency is often not advertised on the outside. One may occasionally see a handwritten note sticked to the shop door "no card payment", or nothing, or the waiter telling you that card payment is (currently?) not possible, some may have a sticker with logos of accepted cards on their door, but not all. Plus, if e.g. google pay is missing from that sticker, it doesn't mean that it is not available, it just means that they didn't put the sticker there (yet).

- **🪧 Advertisement / 🚧 Established tags only:** Shops that accept hype currency such as the various cryptocurrencies or start-up payment processors are usually so few when compared to the total number of shops that mapping these at all feels like doing advertising for these

- **💤 No spam:** ... and of course, the answer the user gives will almost always be the same: no.

- **💵 Valuable surveyors:** Hype currency aside, every shop uses some kind of payment processor that accepts certain services, which may change over time. The number of payment processors in an area is much smaller than the number of shops. I find it questionable if tagging all payment services accepted _on each shop_ is the right approach here at all as the tagged information becomes wrong as soon as the payment processor used by the shop changes which services are accepted or the shop changes the payment processor. I am not advocating for tagging the payment processor instead, because this is even less 🕵️👨‍💻 publicly accessible, 

- **🤷 Useful purpose:** To be useful, the data has to have a certain coverage and needs to be kept up to date. However, I strongly doubt there will ever be a good enough coverage that the data will be useful because frankly, the effort involved in keeping this data correct and up to date is just too much. Maybe not if there was just one or two payment services on earth, but...

- **💵 Valuable surveyors:** It is a bottomless pit. There is an ever-growing mass of payment services out there, tagging them all and _keeping them up-to-date_ in OpenStreetMap is unrealistic. At most, what would be realistic is to make the distinction "card payments" / "cash payments", without specifying which kind of cards - cause that is rather dependent of the region you are in right now than the specific shop you are in. However, looking at taginfo, there is not really any single established value for specifying that unspecified "cards" are accepted

**Comment by FloEdelmann** [comment]
(see also https://github.com/streetcomplete/StreetComplete/issues/5097#issuecomment-1609335274)

**Comment by squeak9** [comment]
Gotcha, thanks for the detailed response.

Keep it up with this fantastic app. 👍
